### PR TITLE
chore: 提交ExternalStoragePersistServiceImpl单元测试

### PR DESCRIPTION
**Pull request by kangzemin**
提交类ExternalStoragePersistServiceImpl的单元测试

### CODE REVIEW

**Comment by brotherlu-xcq** [comment]
Hi, @kangzemin you must assign the CLA first.

File: config/pom.xml [hunk 1]
```diff
@@ -16,25 +16,25 @@
   -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    
+
```
Comment (KomachiSion) [inline config/pom.xml:19]: Please do not change the indent. Please use nacosCodeStyle to reformat codes.

**Review by KomachiSion — CHANGES_REQUESTED** [review]

**Comment by KomachiSion** [comment]
Please fix the IT, thanks.

**Comment by KomachiSion** [comment]
CI can't pass and author has no more response and mofidied. Close PR.

**Pull request closed without merging**